core/src/main/java/io/grpc/PickFirstBalancerFactory.java

```diff
@@ -120,6 +120,11 @@ public void shutdown() {
         subchannel.shutdown();
       }
     }
+
+    @Override
+    public String toString() {
+      return "PickFirstBalancer";
```

Comment:
If we don't define this explicitly, then we can differentiate between different instances. Would that be something we want?

If we really only want to show only the class name, can we get it via something like `getClass().getSimpleName()`?

Reply:
Done.